packages/logging/src/loggers/console.logger.ts

```diff
@@ -105,25 +107,51 @@ export class ConsoleLogger implements LoggerInterface {
     this.log(log);
   }
 
+  private setupStackedLogsArrayIfRequired(traceId?: string) {
+    if(this.stackedLogs.hasOwnProperty(CommonModuleKeyname) === false) {
+      this.stackedLogs[CommonModuleKeyname] = [];
+    }
+
+    if(traceId && this.stackedLogs.hasOwnProperty(traceId) === false) {
+      this.stackedLogs[traceId] = [];
+    }
+  }
+
   private addStackedLog(log: LogModel) {
+    const stackedLogsKey = log.traceId ?? CommonModuleKeyname;
+
     // Push the log at the end of the array
-    this.stackedLogs.push(log);
+    this.stackedLogs[stackedLogsKey].push(log);
 
     // If the stacked logs is already at the maximum number of logs, we delete the first log.
-    if (this.stackedLogs.length >= this.numberOfStackedLogs) {
-      this.stackedLogs.shift();
+    if (this.stackedLogs[stackedLogsKey].length >= this.numberOfStackedLogs) {
```

Comment:
On prend pas en compte le nombre qui serait log par les common ? Faudrait peut etre clear ?
`this.stackedLogs[CommonModuleKeyname] = []`